### PR TITLE
also build against IC-2021.2

### LINKED ISSUE #244: doesn't compile with IC-2021.2
```
> Task :compileKotlin
e: Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to suppress errors
w: Some JAR files in the classpath have the Kotlin Runtime library bundled into them. This may cause difficult to debug problems if there's a different version of the Kotlin Runtime library in the classpath. Consider removing these libraries from the classpath
w: /home/runner/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2021.2/b0727ceddea2b62b16825db9308e14a470198e7f/ideaIC-2021.2/lib/kotlin-stdlib-jdk8.jar: Library has Kotlin runtime bundled into it
e: /home/runner/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2021.2/b0727ceddea2b62b16825db9308e14a470198e7f/ideaIC-2021.2/lib/kotlin-stdlib-jdk8.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.16.
e: /home/runner/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2021.2/b0727ceddea2b62b16825db9308e14a470198e7f/ideaIC-2021.2/lib/kotlin-stdlib-jdk8.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.16.
e: /home/runner/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2021.2/b0727ceddea2b62b16825db9308e14a470198e7f/ideaIC-2021.2/lib/kotlin-stdlib-jdk8.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.16.
```

### PR DESCRIPTION
**Pull request by adietish**
fixes #244

### CODE REVIEW

**Review by jeffmaury — CHANGES_REQUESTED** [review]
Thank you Kotlin

**Comment by adietish** [comment]
@jeffmaury now building with different kotlin artifacts starting with IC-2020.3 (build 203). I tested the binary build with 2019.3 in IC-2021.2. Works fine. Please re-review.

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+     * Returns sinceBuild for a given IC- version.
+     *
+     * Supported cases:
+     * (1) Year and Version -- (a) IC-2017.3, (b) 2017.2.4, (c) 2018.1
+     * (2) Build ID -- (a) IU-163.7890, (b) 171.1234
+     * (3) Named Snapshots -- (a) LATEST-EAP-SNAPSHOT, (b) LATEST-TRUNK-SNAPSHOT
+     *
+     * Taken and adapted from https://github.com/rhdunn/xquery-intellij-plugin/blob/master/build.gradle#L1-L47
+     */
+    ext.getSinceIdeaBuild = { version ->
```
Comment (jeffmaury) [inline build.gradle:12]: That seems too complex to me if you need to convert ideaVersion to a 3 digit why don't you use substring ?

File: gradle.properties [hunk 2]
```diff
@@ -1,9 +1,17 @@
-ideaVersion=IC-2018.3
+ideaVersion=IC-2019.3
```
Comment (jeffmaury) [inline gradle.properties:1]: Is that your lowest version now ? In that case you need to update IJ.yml

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-all.zip
```
Comment (jeffmaury) [inline gradle/wrapper/gradle-wrapper.properties:3]: I would have it part of another PR unless its required

Comment (adietish) [reply gradle/wrapper/gradle-wrapper.properties]: removed it

File: gradle.properties (hunk 2 above)
Comment (adietish) [reply gradle.properties:1]: removed IC-2018.3 in IJ.yml

File: build.gradle (hunk 1 above)
Comment (adietish) [reply build.gradle]: replaced by a simple transformation ideaVersion->sinceIdeaBuild

**Comment by adietish** [comment]
@jeffmaury corrected the portions you complained about. Please re-review.

**Comment by adietish** [comment]
@jeffmaury IC-2019.1, IC-2019.2 are removed: https://github.com/redhat-developer/intellij-kubernetes/pull/243/files#diff-d2a86296b50b44799992fba964e1e33c7d03d00535fd5e24388d3981f6415332R17. Please re-verify

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**